src/components/Heroes/index.jsx

```diff
@@ -100,7 +100,18 @@ class RequestLayer extends React.Component {
             icon=""
             twoLine
           />
-          <Table data={data} columns={_columns} loading={loading} />
+          <div style={{ position: 'relative' }}>
+            <Table data={data} columns={_columns} loading={loading} />
```

Comment:
Planning to extract these two tables into a single component to hide the implementation details of the fixed column behaviour